src/components/secrets/Secret.tsx

```diff
@@ -654,6 +658,22 @@ export const SecretForm: React.FC<SecretFormProps> = function (props) {
                 })
                 setSecretData(json)
             }
+            if(isESO && Array.isArray(json?.esoData)){
+                let jsonList = json?.esoData.map((j) => {
```

Comment:
same here

Reply:
If possible move to separate function and use at both places

Reply:
Next task this function will be different so not creating common function for this